Dockerfile

```diff
@@ -1,19 +1,20 @@
 FROM ansibleplaybookbundle/apb-base
 
 LABEL "com.redhat.apb.spec"=\
-"dmVyc2lvbjogMS4wCm5hbWU6IG1vYmlsZS1jbGllbnQtc2VydmljZS1hcGIKZGVzY3JpcHRpb246\
-IEFlcm9HZWFyIG1vYmlsZS1jbGllbnQtc2VydmljZSBTZXJ2ZXIgCmJpbmRhYmxlOiBUcnVlCmFz\
-eW5jOiBvcHRpb25hbAptZXRhZGF0YToKICBkaXNwbGF5TmFtZTogTW9iaWxlIENsaWVudCBTZXJ2\
-aWNlIDMKICBsb25nRGVzY3JpcHRpb246ICJBbiBBUEIgZGVwbG95aW5nIHRoZSBBZXJvR2VhciBt\
-b2JpbGUtY2xpZW50LXNlcnZpY2UgU2VydmVyIHdpdGggcGVyc2lzdGVudCBzdG9yYWdlIGFuZCBh\
-dXRoZW50aWNhdGlvbiB0aHJvdWdoIE9wZW5zaGlmdCIKICBpbWFnZVVybDogImh0dHBzOi8vcGJz\
-LnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy8xNzk0NDQwMDA1L2Flcm9nZWFyX2ljb24tMV80MDB4\
-NDAwLnBuZyIKICBkb2N1bWVudGF0aW9uVXJsOiAiaHR0cHM6Ly9kb2NzLmFlcm9nZWFyLm9yZy9l\
-eHRlcm5hbC9hcGIvbW9iaWxlLWNsaWVudC1zZXJ2aWNlLmh0bWwiCiAgcHJvdmlkZXJEaXNwbGF5\
-TmFtZTogIlJlZCBIYXQsIEluYy4iCiAgZGVwZW5kZW5jaWVzOiBbXQogIHNlcnZpY2VOYW1lOiBt\
-b2JpbGUtY2xpZW50LXNlcnZpY2UKICBiaW5kaW5nc0xpbWl0OiAyCnBsYW5zOgogIC0gbmFtZTog\
-ZGVmYXVsdAogICAgZGVzY3JpcHRpb246IFBlcnNpc3RlbnQgZGVwbG95bWVudCBvZiBtb2JpbGUt\
-Y2xpZW50LXNlcnZpY2UKICAgIGZyZWU6IFRydWUKICAgIHBhcmFtZXRlcnM6IFtdCg=="
+"dmVyc2lvbjogMS4wCm5hbWU6IG1jcy1zZXJ2ZXItYXBiCmRlc2NyaXB0aW9uOiBBZXJvR2VhciBt\
+Y3Mtc2VydmVyIFNlcnZlciAKYmluZGFibGU6IFRydWUKYXN5bmM6IG9wdGlvbmFsCm1ldGFkYXRh\
+OgogIGRpc3BsYXlOYW1lOiBNb2JpbGUgQ2xpZW50IFNlcnZpY2UgMQogIGxvbmdEZXNjcmlwdGlv\
+bjogIkFuIEFQQiBkZXBsb3lpbmcgdGhlIEFlcm9HZWFyIG1jcy1zZXJ2ZXIgU2VydmVyIHdpdGgg\
+cGVyc2lzdGVudCBzdG9yYWdlIGFuZCBhdXRoZW50aWNhdGlvbiB0aHJvdWdoIE9wZW5zaGlmdCIK\
+ICBpbWFnZVVybDogImh0dHBzOi8vcGJzLnR3aW1nLmNvbS9wcm9maWxlX2ltYWdlcy8xNzk0NDQw\
+MDA1L2Flcm9nZWFyX2ljb24tMV80MDB4NDAwLnBuZyIKICBkb2N1bWVudGF0aW9uVXJsOiAiaHR0\
+cHM6Ly9kb2NzLmFlcm9nZWFyLm9yZy9leHRlcm5hbC9hcGIvbWNzLXNlcnZlci5odG1sIgogIHBy\
+b3ZpZGVyRGlzcGxheU5hbWU6ICJSZWQgSGF0LCBJbmMuIgogIGRlcGVuZGVuY2llczogCiAgICAg\
+IC0gImRvY2tlci5pby9vcGVuc2hpZnQvb2F1dGgtcHJveHk6djEuMS4wIgogICAgICAtICJkb2Nr\
+ZXIuaW8vYWVyb2dlYXIvbW9iaWxlLWNsaWVudC1zZXJ2aWNlOmxhdGVzdCIKICBzZXJ2aWNlTmFt\
+ZTogbWNzLXNlcnZlcgogIGJpbmRpbmdzTGltaXQ6IDIKcGxhbnM6CiAgLSBuYW1lOiBkZWZhdWx0\
+CiAgICBkZXNjcmlwdGlvbjogUGVyc2lzdGVudCBkZXBsb3ltZW50IG9mIG1jcy1zZXJ2ZXIKICAg\
+IGZyZWU6IFRydWUKICAgIHBhcmFtZXRlcnM6IFtdCg=="
```

Comment:
when I ran `apb build`, the Dockerfile is changed again. Should this file be regenerated?

Reply:
Yes